src/js_stream.cc

```diff
@@ -154,9 +150,8 @@ void JSStream::New(const FunctionCallbackInfo<Value>& args) {
 
 template <class Wrap>
 void JSStream::Finish(const FunctionCallbackInfo<Value>& args) {
-  Wrap* w;
   CHECK(args[0]->IsObject());
-  ASSIGN_OR_RETURN_UNWRAP(&w, args[0].As<Object>());
+  Wrap* w = static_cast<Wrap*>(Wrap::FromObject(args[0].As<Object>()));
```

Comment:
`StreamReq::FromObject()`?  `static_cast<Wrap*>(Wrap::FromObject(...))` probably looks redundant and mildly confusing to the casual reader.